bridge-fungible-token-abi/src/lib.sw

```diff
@@ -1,17 +1,26 @@
 library bridge_fungible_token_abi;
 
-use std::address::Address;
-use std::contract_id::ContractId;
+use std::{contract_id::ContractId, identity::Identity, vm::evm::evm_address::EvmAddress};
 
 abi BridgeFungibleToken {
+    #[storage(read, write)]
+    fn claim_refund(originator: Identity, asset: EvmAddress);
```

Comment:
Should the originator also be an 'EvmAddress'? Or at the very least it should match the 'from' field of the 'MessageData' struct.

Reply:
Yeah, that's a good point. I should make it an `EvmAddress`.
As an aside, I wonder if `EvmAddress` should be a third variant on the `Identity` Enum... 🤔 
Maybe not, as you can't really act on it in the same way directly, so it's a bit different.